packages/harpguru-core/src/hooks/use-sizes/use-sizes.ts

```diff
@@ -78,7 +78,7 @@ export const useSizes = (): SizeSchemes => {
   const legendWidth = columnWidth
   const zoomSlideWidth = useIsZoomedColumnBounds() === false ? 0 : columnWidth
 
-  const { length: groupCount } = useOctaveColumnGroups('harpface1', true)
+  const { length: groupCount } = useOctaveColumnGroups()
```

Comment:
This still isn't perfect. I would like the size scheme to remain the same during all slide positions on a zoom. Currently we will sometimes see fewer or greater numbers of octave boundaries in a single view depending on the position of the zoom. This will lead to a different seed size.

The only way I can think to address this but be safely flexible to all possible presentations within a zoom is to have the number returned from here somehow figure out the worst case scenario (ie the largest number of column boundaries which can be seen at that zoom level for that harp) and use that as the source of the size maths.

Reply:
Done